configs/benchmark-stn.json

```diff
@@ -1,21 +1,21 @@
 {
-  "description": "aws 510 nodes",
+  "description": "aws 160 nodes, 4 shards",
   "client": {
     "num_vm": 0,
-    "type": "t3.medium",
+    "type": "m5a.large",
     "regions": "iad"
   },
   "leader": {
-    "num_vm": 1,
+    "num_vm": 4,
```

Comment:
num_vm: 1, as we have 4 regions already, num_vm is per region number

Reply:
oh okay, changed to 1